### PR TITLE
Added banned items per 2022-03-25 update

### PR DESCRIPTION
**Pull request by markXLIX**
https://www.fcc.gov/supplychain/coveredlist

### CODE REVIEW

**Review by GaryGapinski — CHANGES_REQUESTED** [review]
The assertion` inventory-item-has-prohibited-vendor-name` message is incorrect for a successful test outcome (i.e., it does not conform to the acceptance criterion "All Schematron assertion messages are declarative statements which affirm the positive test outcome.").

The assertion id itself does not follow the same manner as (most) other assertions, namely, stating the positive outcome, but that can be relegated to a separate issue to fix all such "negative" identifiers and need not be corrected here.

File: src/validations/rules/ssp.sch [hunk 1]
```diff
@@ -2175,22 +2175,22 @@
...
                 'Hikvision', 'Hangzhou Hikvision', 
                 'Huawei', 'Huawei Technologies Company', 
                 'HyTera', 'Hytera Communications Corporation', 
                 'AO Kaspersky Lab', 'Kaspersky Lab', 'Kaspersky', 
                 'ZTE', 'ZTE Corporation', 
-                'China Mobile International USA Inc',
-                'China Telecom (Americas) Corp'"/>
+                'China Mobile', 'China Mobile International USA Inc',
+                'China Telecom', 'China Telecom (Americas) Corp')"/>
             <sch:assert
```
Comment (GaryGapinski) [inline src/validations/rules/ssp.sch:2187]: This assertion fails to exhibit a correct message for a postive test outcome.

The acceptance criterion is "**All Schematron assertion messages are declarative statements which affirm the positive test outcome.**".

The message you are proposing is "**The information system contains a banned vendor -  '<xsl:value-of select="o:prop[@name eq 'vendor-name']/@value"/>'.  See FAR 889(a)(1)(B)**.". When this assertion test - `not(o:prop[@name eq 'vendor-name']/@value = $prohibit-vendor)` - i.e., encounters an `<inventory-item>` `<prop name="vendor">` which is not a banned value, is positive (boolean true) for any `<inventory-item>` the Schematron result (SVRL output) will have the assertion message citing the value as banned when it is not.

When the assertion is successful, the message will be incorrect and additionally cite the correct vendor-name value as being a banned one.

A sample positive assertion message would be something like "_The inventory item does not cite a banned vendor._".

Let's discuss this tomorrow if questions remain.

**Comment by markXLIX** [comment]
closes #403

**Review by GaryGapinski — APPROVED** [review]
Looks good.

I am unsure you can put a "closes \#nnn" in a comment as opposed to the issue description, but try it and see.

**Comment by markXLIX** [comment]
That was my thought about the 'closes ####' but I figured it couldn't hurt to try it.

**Pull request merged**